resources/create_modulefile.rb

```diff
@@ -0,0 +1,34 @@
+resource_name :create_modulefile
```

Comment:
Could you add a description of what is a modulefile and for what are we using it?

Reply:
Added!